admin-manual/installation/installation.rst

```diff
@@ -6,290 +6,245 @@ Installation
 
 *On this page*
 
+* :ref:`Overview <overview>`
 * :ref:`Technical requiremnets <tech-requirements>`
+* :ref:`New Installation <new-install>`
+* :ref:`Upgrading from 1.5 <upgrade>`
+* :ref:`Advanced <advanced>`
 
-* :ref:`Recommended Minimum Requirements for small-scale testing <requirements-small>`
-
-* :ref:`Recommended Minimum Requirements for production processing <requirements-production>`
-
-* :ref:`Install from packages <install-packages>`
+.. _overview:
 
-* :ref:`Install new source <install-source>`
+Overview
+========
 
-* :ref:`Install for CentOS/Redhat <centos>`
+Archivematica is not a single application, there are a dozens of different 
+components and tools required for a full working installation.  As a result 
+there are many possible deployment configurations.  
 
-* :ref:`Upgrade from Archivematica 1.5 for CentOS/Redhat <centos-upgrade>`
+These instructions are designed to get you up and running as quickly as 
+possible, even if you are not familiar with the linux operating system or the 
+various tools and applications that are bundled into an Archivematica 
+installation.  
 
-* :ref:`Docker <docker>`
+Experience with the linux command line is helpful, and to support a running 
+production system, it should be considered a requirement.  
 
-* :ref:`Install for development <development>`
-
-* :ref:`Advanced <advanced>`
+Don't be afraid to ask for help if you get stuck, or if you don't understand
+some part of the instructions.  The Archivematica google group is a good place
+to look for assistance.
```

Comment:
Suggest linking the google group, `Archivematica Google group <https://groups.google.com/forum/#!forum/archivematica>`